src/open_company_auth/app.clj

```diff
@@ -49,35 +52,26 @@
   (GET "/slack-oauth" {params :params} (oauth-callback slack/oauth-callback params))
   (GET "/test-token" [] (jwt-debug-response test-token)))
 
-(defonce ^:private hot-reload-routes
-  ;; Reload changed files without server restart
-  (if config/hot-reload
-    (wrap-reload #'auth-routes)
-    auth-routes))
+(timbre/merge-config!
+  {:appenders {:spit (appenders/spit-appender {:fname "/tmp/oc-auth.log"})}})
 
-(defonce ^:private cors-routes
-  ;; Use CORS middleware to support in-browser JavaScript requests.
-  (wrap-cors hot-reload-routes #".*"))
-
-(defonce ^:private sentry-routes
-  ;; Use sentry middleware to report runtime errors if we have a raven DSN.
-  (if config/dsn
-    (wrap-sentry cors-routes config/dsn)
-    cors-routes))
-
-(defonce app
-  (wrap-params sentry-routes))
+(def app
+  (cond-> #'auth-routes
+    config/hot-reload wrap-reload
+    true              wrap-params
+    true              (wrap-cors #".*")
+    config/dsn        (sentry-mw/wrap-sentry config/dsn)))
 
 (defn start
   "Start a server"
   [port]
   (run-server app {:port port :join? false})
-    (println (str "\n" (slurp (clojure.java.io/resource "./open_company_auth/assets/ascii_art.txt")) "\n"
-      "OpenCompany Auth Server\n"
-      "Running on port: " port "\n"
-      "Hot-reload: " config/hot-reload "\n"
-      "Sentry: " config/dsn "\n\n"
-      "Ready to serve...\n")))
+  (println (str "\n" (slurp (io/resource "ascii_art.txt")) "\n"
+                "OpenCompany Auth Server\n"
+                "Running on port: " port "\n"
+                "Hot-reload: " config/hot-reload "\n"
+                "Sentry: " config/dsn "\n\n"
+                "Ready to serve...\n")))
```

Comment:
It'd be good @martinklepsch to add the S3 bucket being used to the startup output.